i-am-terrible.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+// I'm writing this directly in the github gui, so it should be full of syntax errors, in addtion to these terrible ones I'm making on purpose      
+
+function IAMAFUNCTION(int $one) {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->